src/App.vue

```diff
@@ -17,10 +17,10 @@ div#wrapper
   div.container.aw-container
     // TODO: Refactor into Mainmenu component
     b-nav.row.aw-navbar
-      b-nav-item(to="/")
+      b-nav-item(to="/" exact v-on:click="current='home'" :class="{ inCategory: current=='home' || isHomePath}"  )
         icon(name="home")
         | Home
-      b-nav-item(v-if="activity_hosts.length === 1", v-for="host in activity_hosts", :key="host", :to="'/activity/' + host")
+      b-nav-item(v-if="activity_hosts.length === 1", v-for="host in activity_hosts", :key="host", :to="'/activity/' + host" v-on:click="current='activity'" :class="{ inCategory: current=='activity' || isActivityPath}")
```

Comment:
This `v-on:clock="current='x'"` seems like it would be buggy to me.
What if you for example follow a link directly to activity without actually pressing the "Activity" button?
Why doesn't "isActivityPath" suffice?

Reply:
"isActivityPath" does suffice. I posted the new commits for the updated version